### PR TITLE
Initial checkin of logical replication scripts

### PR DESCRIPTION
**Pull request by maheese**
These are the scripts that are used to set up logical replication between a Cloud.gov EASEY database and a database in the DMAP environment.

### CODE REVIEW

File: postgres/logical-replication/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+**See this link for more information on why the heartbeat process is needed [https://wolfman.dev/posts/pg-logical-heartbeats/](https://wolfman.dev/posts/pg-logical-heartbeats/)*
+
+## DMAP
+The DMAP script creates the subscription to the data in the Cloud.gov database with the user set up above
+
+## Deployment
+The high level steps to set up logical replication between the two databases are as follows:
+
+- Logical replication is not enabled by default in the Cloud.gov AWS RDS databases.  To get the feature enabled, send an email to Cloud.gov support providing the information for the target database.
```
Comment (j-tafoya) [inline postgres/logical-replication/README.md]: And inform them to restart the database for the change to take effect.

File: postgres/logical-replication/sql/cloudgov/4_create_heartbeat.sql [hunk 2]
```diff
@@ -0,0 +1,15 @@
+--Creates the heartbeat process to keep the WAL files from piling up on the db server
+--See https://wolfman.dev/posts/pg-logical-heartbeats/ for a detailed explaination
+DROP TABLE IF EXISTS camdaux._heartbeat;
```
Comment (j-tafoya) [inline postgres/logical-replication/sql/cloudgov/4_create_heartbeat.sql]: Is this needed when we have the pg_stat_replication cron job?

**Review by j-tafoya — COMMENTED** [review]
I can't remember if we decided we still needed the heartbeat if we were writing the pg stat logs?

File: postgres/logical-replication/README.md (hunk 1 above)
Comment (maheese) [reply postgres/logical-replication/README.md]: I added a note about the restart to the readme.

File: postgres/logical-replication/sql/cloudgov/4_create_heartbeat.sql (hunk 2 above)
Comment (maheese) [reply postgres/logical-replication/sql/cloudgov/4_create_heartbeat.sql]: You are correct.  I've removed the heartbeat process and added notes/comments to indicate that the pg_stat_replication job serves two purposes and should not be disabled.

**Pull request merged**